apps/api/src/app/user/user.controller.ts

```diff
@@ -84,4 +88,22 @@ export class UsersController {
       })
     );
   }
+
+  @Put('/onboarding-tour')
+  @ApiResponse(UserOnboardingTourRequestDto)
+  @ApiOperation({
+    summary: 'Update onboarding tour count',
+  })
+  @ExternalApiAccessible()
```

Comment:
Can we hide this API From the spec? Not sure it's needed there and can confuse people I think. wdyt?

Reply:
Agree, hiding it